mllib/src/main/scala/org/apache/spark/mllib/fpm/AssociationRules.scala

```diff
@@ -142,5 +142,14 @@ object AssociationRules {
     def javaConsequent: java.util.List[Item] = {
       consequent.toList.asJava
     }
+
+    /**
+     * Returns String, [antecedent] => [consequent]: [confidence]
```

Comment:
Trivial, but you should use `@return` and there's no need to state the return type again. Remove the blank line below too